tests/app/test_modifier.py

```diff
@@ -5,6 +5,8 @@
 from flask import current_app
 from flask import g
 from sqlalchemy import create_engine
+from sqlalchemy.engine import Engine
+from sqlalchemy import event
```

Comment:
With the imports, you can add them on the same line......  `from sqlalchemy import create_engine, event`